tools/test/TestCSVFileImport.cc

```diff
@@ -53,3 +53,27 @@ TEST (TestCSVFileImport, test10rows) {
   EXPECT_EQ(expected, output);
   EXPECT_EQ("", error);
 }
+
+TEST (TestCSVFileImport, testTimezoneOption) {
```

Comment:
This test case succeeds without your patch, `CSVFileImport.cc`.

Reply:
I test the case again when I delete the line"options.setTimezoneName(timezoneName);" in CSVFileImport.cc. The case is failed. The error message show
"Expected: expected
      Which is: "{\"_a\": \"2021-12-26 16:00:00.0\"}\n"
To be equal to: output
      Which is: "{\"_a\": \"2021-12-27 00:00:00.0\"}\n"